### ISSUE TITLE
Add a fieldtype to select a source or feed

### ISSUE THREAD

**Issue by bartrylant** [verbb/social-feeds#8]
### What are you trying to do?

I'd like to use this plugin on a site where every event has its own feed (based on twitter hashtags). Therefore I need to make a new source with the hashtag, but I'd like the content editors to be able to select the correct source/feed and link it to their entry.

### What's your proposed solution?

A source/feed fieldtype that lists all the available sources/feeds. Content editor can relate the source/feed to an entry.

**Comment by engram-design** [comment]
Just checking if it would be fine if these were two different field types - one for a source, and another for a feed? They aren't elements, so the UI would need to be a dropdown. This also begs the question - are you okay with it being a singular source/feed?

**Comment by bartrylant** [comment]
Yes, two field types and a single source/feed would be fine. I could also put it in a Matrix field if I really need more.

Actually I didn't really check the docs before I installed the plugin as a trial -- I like seeing how things work. I was expecting this to be a field type, not a section in the control panel. My client often has news with a particular hashtag, and would like to attach X/instagram posts with that tag on that entry. So I though you would be able to just throw in some hashtags in a field, without having to make a feed/source first in a different part of the control panel. I thought it would make sense to do it like that.

**Comment by engram-design** [comment]
Yeah, that's an interesting use-case. Certainly something we can lean towards in the future by making that a nice experience with a field. The primary purpose (right now) is about having a consolidated collection of posts, and they need to be defined in your feed, that pulls in sources from socials.

Having that be on-demand might be tricky, but something we can work on.
